.github/workflows/bats.yaml

```diff
@@ -0,0 +1,86 @@
+name: BATS on macOS on latest CI build
+on:
+  workflow_dispatch:
+    inputs:
+      owner:
+        description: Override owner (e.g. rancher-sandbox)
+        type: string
+      repo:
+        description: Override repository (e.g. rancher-desktop)
+        type: string
+      branch:
+        description: Override branch (e.g. main)
+        type: string
+      tests:
+        description: 'Tests (e.g. tests/containers/*)'
+        default: 'tests/*'
+        type: string
+      results:
+        description: Results
```

Comment:
Clarify a bit more the description of Results e.g "Number of workflows to query for a Package"